src/main/resources/data/pieces/268758-02_created_by_holding.json

```diff
@@ -0,0 +1,11 @@
+{
+  "id": "774608cf-6e7f-498a-b0C6-3bd38d8b8f9e",
+  "caption": "Tutorial Volume 25",
+  "comment": "The best Edition",
+  "format": "Physical",
+  "itemId": "d6f7c1ba-a237-465e-94ed-f37e91bc64bd",
```

Comment:
This item is for `7fbd5d84-62d1-44c6-9c45-6cb173998bbd` instance which is linked to `774608cf-6e7f-498a-b0C6-3bD38D8b8F9e` PO Line

Reply:
itemId should not be here because the poLine associated with this piece have `createInventory="Instance"`
The same for a [piece](https://github.com/folio-org/mod-orders-storage/blob/MODORDSTOR-63_add_sample_data_for_pieces/src/main/resources/data/pieces/1a902141-d8c6-4cf3-8a19-193eca007478.json) with id=1a902141-d8c6-4cf3-8a19-193eca007478